### PR TITLE
feat: adjusted messaging on label pointers

### PR DESCRIPTION
**Pull request by frosso**
## Description

Adjustments on the admin "pointers"/"popovers" displayed on the orders page.

### Related issue(s)

Closes https://github.com/Automattic/woocommerce-services/issues/2196

### Steps to reproduce & screenshots/GIFs

In order to see the "new user" pointer with adjusted messaging:
- PLEASE MAKE SURE IT IS A BRAND NEW SITE - there should be no previously created labels/no pointers displayed
- Create a new order
- Destination can be domestic or international
- See pointer in order management view
- Feel free to change the DHL flag(s), to ensure that the messaging is adjusted

In order to see the "print dhl labels now!" pointer:
- PLEASE MAKE SURE IT IS AN EXISTING SITE - labels should have been created and the updated version of the plugin should not have run
- Go to an existing order
- Make sure order destination is international (not domestic)
- Make sure site's DHL flag is enabled
- Make sure you have the most updated service data (or refresh it in **WooCommerce > Status > WooCommerce Services**)
- See pointer in order management view

New user, DHL is disabled: shoes "new user" popover with old messaging
Screen Shot 2020-09-24 at 1 32 46 PM

New user, DHL is enabled: shows "new user" popover with adjusted messaging
**PLEASE NOTE**: I am not checking the order's destination, just whether the site has DHL enabled
Screen Shot 2020-09-24 at 12 18 57 PM

Existing user, eligible order: shows new popover
Screen Shot 2020-09-24 at 9 25 14 AM

Existing user, order is not eligible (because of domestic destination): do not show popover
**PLEASE NOTE**: message might also not be displayed because of the individual site flag
Screen Shot 2020-09-24 at 9 39 10 AM

### Checklist

- [ ] unit tests
I didn't see existing tests for pointers - any recommendations?

- [x] `changelog.txt` entry added

### CODE REVIEW

**Comment by harriswong** [comment]
👀

File: classes/class-wc-connect-nux.php [hunk 1]
```diff
@@ -146,24 +147,61 @@ public function register_order_page_labels_pointer( $pointers ) {
...
+				return $pointers;
+			}
+
+			$pointers[] = array(
+				'id' => 'wc_services_new_carrier_dhl_express',
+				'target' => '#woocommerce-order-label .button',
+				'options' => array(
+					'content' => sprintf( '<h3>%s</h3><p>%s</p>',
+						__( 'Discounted DHL Shipping Labels' ,'woocommerce-services' ),
+						__( "WooCommerce Shipping now supports DHL. Purchase and print discounted labels from DHL and USPS right here.", 'woocommerce-services' )
```
Comment (harriswong) [inline classes/class-wc-connect-nux.php:198]: @brentmackinnon made a suggestion [here](https://github.com/Automattic/woocommerce-services/issues/2196#issuecomment-700892675).  Let's make sure we have the right copy.

**Review by harriswong — APPROVED** [review]
LGTM 👍 
Just one comment on the copy of the pop up message.

I tested 
[new user , old user] x [ flags ]

File: classes/class-wc-connect-nux.php (hunk 1 above)
Comment (frosso) [reply classes/class-wc-connect-nux.php:198]: I confirmed the messaging, thanks for the reminder! 💯 

```suggestion
						__( "WooCommerce Shipping now supports DHL Express labels for international shipments. Purchase and print discounted labels from DHL and USPS right here.", 'woocommerce-services' )
```

Comment (harriswong) [reply classes/class-wc-connect-nux.php]: Should we use `DHL Express` throughout all copy or a mix of `DHL` and `DHL Express`?

Comment (frosso) [reply classes/class-wc-connect-nux.php]: I confirmed with Brent to remove "Express"

File: classes/class-wc-connect-nux.php [hunk 2]
```diff
@@ -146,24 +147,61 @@ public function register_order_page_labels_pointer( $pointers ) {
...
+				return $pointers;
+			}
+
+			$pointers[] = array(
+				'id' => 'wc_services_new_carrier_dhl_express',
+				'target' => '#woocommerce-order-label .button',
+				'options' => array(
+					'content' => sprintf( '<h3>%s</h3><p>%s</p>',
+						__( 'Discounted DHL Shipping Labels' ,'woocommerce-services' ),
+						__( "WooCommerce Shipping now supports DHL Express labels for international shipments. Purchase and print discounted labels from DHL and USPS right here.", 'woocommerce-services' )
```
Comment (frosso) [inline classes/class-wc-connect-nux.php:198]:
```suggestion
						__( "WooCommerce Shipping now supports DHL labels for international shipments. Purchase and print discounted labels from DHL and USPS right here.", 'woocommerce-services' )
```

**Pull request merged**